pallets/teerex/src/lib.rs

```diff
@@ -102,6 +102,15 @@ pub mod pallet {
 	#[pallet::getter(fn allow_sgx_debug_mode)]
 	pub type AllowSGXDebugMode<T: Config> = StorageValue<_, bool, ValueQuery>;
 
+	#[pallet::type_value]
+	pub fn HeartbeatTimeoutDefault() -> u64 {
+		172_800_000
+	} // 48h
+	#[pallet::storage]
+	#[pallet::getter(fn heartbeat_timeout_storage)]
+	pub type HeartbeatTimeoutStorage<T: Config> =
```

Comment:
I think `HeartbeatTimeout` is enough - it's a storage already :)

Reply:
👍 make sense.